spec/features/invalid_user_visits_newstop_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+feature 'invalid user is blocked from new page'  do
+
+  user = User.create(
+    user_name: "SadPath",
+    email: "sad@path.com",
+    password: "password"
+  )
+
+  scenario 'user sees not found error' do
+    expect{visit new_stop_path}.to raise_error
+  end
+
+end
```

Comment:
Come on my man, you know that this could be a bit better. Please dont try to sneak this in! 
This test should exist in your `new` page spec, not in its own file.

Additionally you should be checking that the user sees a 404 page. Be a little bit more verbose with this one. Take your time. Make sure that this ISNT happening for a logged in user.